docs/install_guides/installing-openvino-docker-windows.md

```diff
@@ -83,38 +103,39 @@ docker run -it --rm <image_name>
 If you want to try some demos then run image with the root privileges (some additional 3-rd party dependencies will be installed):
```

Comment:
Can you switch this line to the text below? Open Model Zoo demos are no longer included in the images since 2022.1.
> If you want to try some samples, run the image with the following command

Reply:
updated